core/source/org/libreoffice/ide/eclipse/core/editors/pack/PackagePropertiesEditor.java

```diff
@@ -158,8 +154,8 @@ public boolean isDirty() {
     @Override
     public void doSave(IProgressMonitor pMonitor) {
         try {
-            mModel.write();
-            mSourcePage.doRevertToSaved();
+            writeToSource(mModel.write());
+            //mSourcePage.doRevertToSaved();
```

Comment:
Please remove unused code instead of commenting it out.

Reply:
I remove it but I don't know what this piece of code is for? On the other hand, if I leave it, we can no longer save correctly (the file still appears unsaved)